chunked.c

```diff
@@ -0,0 +1,121 @@
+/*
+  COSC 6060
+  Matthew Dupont and Louis Finney
+  Assignment 2 - MPI work distribution
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <mpi.h>
+
+#include "polynomial.h"
```

Comment:
I should probably just get rid of this at this point. Was trying to be clever, but honestly beyond the needs of this project.